### ISSUE TITLE
Misaligned road tile

### ISSUE THREAD

**Issue by FlyingSpaceRhino** [CleverRaven/Cataclysm-DDA#74730]
### Describe the bug

I found a single tile of road that was facing the wrong direction, disconnected from the rest of the road.

### Attach save file

[New England-trimmed.tar.gz](https://github.com/user-attachments/files/15942903/New.England-trimmed.tar.gz)

### Steps to reproduce

1. Load save, look at it

### Expected behavior

Road tiles should be connected.

### Screenshots

![Screenshot (2186)](https://github.com/CleverRaven/Cataclysm-DDA/assets/38295275/54a75a34-296e-483c-8295-0354fda418e6)
![Screenshot (2188)](https://github.com/CleverRaven/Cataclysm-DDA/assets/38295275/9db8fa01-6058-4434-aac2-9ee95150858f)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.4529 (22H2)
- Game Version: cdda-experimental-2024-06-23-0432 9d91f03 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by ZhilkinSerg** [comment]
Is it possible that map could've been generated in older version?

**Comment by FlyingSpaceRhino** [comment]
> Is it possible that map could've been generated in older version?

No. This was a brand new save on a brand new download of 9d91f03

**Comment by PatrikLundell** [comment]
It COULD be caused by the rotation issued fixed by #74732, which hasn't been approved yet. I wasn't aware of the road rotation issue, however.
